inc/compatibility/easy_digital_downloads.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Author:          Uriahs Victor
+ * Created on:      14/07/2021 (d/m/y)
+ *
+ * @package Neve
+ */
+namespace Neve\Compatibility;
+
+/**
+ * Class Easy_Digital_Downloads
+ *
+ * @package Neve\Compatibility
+ */
+class Easy_Digital_Downloads {
+
+	/**
+	 * Function that is run after instantiation.
+	 *
+	 * @return void
+	 */
+	public function init() {
+		if ( ! class_exists( 'Easy_Digital_Downloads' ) ) {
+			return;
+		}
+		add_action( 'wp_enqueue_scripts', array( $this, 'neve_dequeue_edd_styles' ) );
+		add_filter( 'edd_settings_styles', array( $this, 'neve_edd_settings_styles' ) );
```

Comment:
These methods can be unprefixed, as they are both inside a class and under the `Neve` namespace 🚀